pkg/controller/operators/olm/requirements.go

```diff
@@ -238,7 +260,7 @@ func (a *Operator) requirementAndPermissionStatus(csv *v1alpha1.ClusterServiceVe
 		return false, nil, fmt.Errorf("could not cast install strategy as type %T", strategyDetailsDeployment)
 	}
 
-	reqMet, reqStatuses := a.requirementStatus(strategyDetailsDeployment, csv.GetAllCRDDescriptions(), csv.GetOwnedAPIServiceDescriptions(), csv.GetRequiredAPIServiceDescriptions())
+	reqMet, reqStatuses := a.requirementStatus(strategyDetailsDeployment, csv.GetAllCRDDescriptions(), csv.GetOwnedAPIServiceDescriptions(), csv.GetRequiredAPIServiceDescriptions(), csv.Spec.NativeAPIs)
 
```

Comment:
Does the GUI ever make use of the previous getter methods? I'm just wondering if it's worth making one for the Native APIs to return an alphabetical listing (or deduplication) like the others.

Reply:
I don't understand how the frontend app would call into the Operator Go source...?

Reply:
I didn't mean call directly. But based on your response there's zero coupling.